charts/common/templates/_helpers.tpl

```diff
@@ -156,6 +157,9 @@ livenessProbe:
       {{- else }}
       cpu: {{ .postgres.cpu }}
       {{- end }}
+      {{- if .postgres.memoryLimt }}
+      memory: "{{ .postgres.memoryLimit }}Mi"
+      {{- else }}
       memory: "{{ .postgres.memory }}Mi"
```

Comment:
missing `{{- end }}`

If you run `helm lint` you will get an error because of this.